grails-app/views/location/list.gsp

```diff
@@ -107,6 +108,18 @@
                                             </td>
                                             <td class="left middle"><format:metadata obj="${locationInstance?.locationType}"/></td>
                                             <td class="left middle">${locationInstance?.locationGroup?:warehouse.message(code:'default.none.label')}</td>
+                                            <td class="middle">
+                                                <g:if test="${locationInstance?.status == LocationStatus.ENABLED }">
+                                                    <span class="active">
```

Comment:
This could be as simple as:
```
<span class="${locationInstance?.status == LocationStatus.ENABLED ? 'active' : 'inactive' }">
    <format:metadata obj="${locationInstance?.status}"/>
</span>
```